command/agent/template/template.go

```diff
@@ -246,31 +246,23 @@ func (ts *Server) Run(ctx context.Context, incoming chan string, templates []*ct
 				ts.runner.Stop()
 				return nil
 			}
-		default:
-			// We are using default instead of a new case block to prioritize the case where <-incoming has a new value over
```

Comment:
Does the new code break the intentions of the author? If I'm understanding the author correctly, `ts.runner.ErrCh` may get an error related to a bad token and the structure of the select statements was meant to give the child priority. This didn't work since it was continuing.